app/templates/buyers/brief_overview.html

```diff
@@ -255,8 +276,8 @@ <h2 class="instruction-list-item-body">
       </ol>
     </div>
     <div class="column-one-third">
-      {% if brief.publishedAt %}
-        {% include 'buyers/_requirements_meta.html' %}
+      {% if brief.status == 'closed' and framework.status in ['live', 'expired'] %}
```

Comment:
You probably don't need to do the framework status check. If a brief has been published then the framework is either in a live or expired state (https://alphagov.github.io/digitalmarketplace-manual/framework-lifecycle-for-developers.html). Therefore this check is not actually doing anything.